src/types/registration.types.ts

```diff
@@ -286,6 +286,40 @@ export interface RegistrationSearchItem {
   };
 }
 
+export interface RegistrationSearchItem2 {
+  recordMetadata: {
+    status: RegistrationStatus;
+    createdDate: string;
+    modifiedDate: string;
+    publishedDate?: string;
+  };
+  id: string;
+  identifier: string;
+  type: PublicationInstanceType | '';
```

Comment:
Kan egentlig denne egentlig være tom?

```suggestion
  type: PublicationInstanceType;
```

Reply:
Tror problemet her ligger i `convertToRegistrationSearchItem` i `registration-helpers`. Da den tar inn en registration, så er arver den det derifra?

Reply:
Æsj, ser det er en liten kødd der ja. Jaja, får være